arviz/plots/bpvplot.py

```diff
@@ -0,0 +1,294 @@
+"""Bayesian p-value Posterior/Prior predictive plot."""
+import numpy as np
+from matplotlib.colors import to_hex
```

Comment:
we can use `az.plots.plot_utils.vectorized_to_hex`. It basically calls `matplotlib.colors.to_hex` but works with lists of colors too (not relevant here) and we could at some point (far future) make a version of the function not requiring matplotlib, i.e. try importing from matplotlib and if not possible use bokeh version (as far as I know bokeh has no function that does this yet)